linera-execution/tests/test_execution.rs

```diff
@@ -1174,45 +1204,57 @@ async fn test_open_chain() -> anyhow::Result<()> {
         ValidatorPublicKey::test_key(0),
         AccountPublicKey::test_key(0),
     )]);
-    let committees = BTreeMap::from([(Epoch::ZERO, committee)]);
+    let committees = BTreeMap::from([(
+        Epoch::ZERO,
+        bcs::to_bytes(&committee).expect("serializing a committee should not fail"),
```

Comment:
It's fine to just `unwrap()` in tests. And this one even returns a result anyway:

```suggestion
        bcs::to_bytes(&committee)?,
```